src/App.js

```diff
@@ -1,12 +1,60 @@
-import React from 'react';
 import './App.css';
+import axios from "axios";
+import {useEffect, useState} from "react";
+import Card from "./components/Card";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    const [pokeList, setPokeList] = useState({})
+    const [link, setLink] = useState('https://pokeapi.co/api/v2/pokemon')
+
+    function handleClick(link) {
+        setLink(link);
+    }
+
+
+    useEffect(() => {
+        async function getTwentyPokes() {
+            try {
+                const fetchDataTwentyPokes = await axios.get(link)
+                console.log(fetchDataTwentyPokes.data)
+                setPokeList(fetchDataTwentyPokes.data)
+            } catch (e) {
+                console.error(e)
+            }
+        }
+
+        getTwentyPokes();
+
+    }, [link])
+
+
+    return (
+        <div className="outer-container">
+            <div className="inner-container">
+                <h1>POKEMON</h1>
+                <div className="button-box">
+                    <button
+                        disabled={pokeList.previous === null}
+                        onClick={(() => {
+                            handleClick(pokeList.previous)
+                        })}>Prev
+                    </button>
+                    <button onClick={(() => {
+                        handleClick(pokeList.next)
+                    })}>Next
+                    </button>
+                </div>
+                {Object.keys(pokeList).length > 0 &&
```

Comment:
Dan kun je deze check wat versimpelen
```suggestion
                {pokeList &&
```